lib/axios.ts

```diff
@@ -28,8 +28,17 @@ export default async function(step: any, config) {
 }
 */
 
-export default async function(step: any, config) {
+export default async function(step: any, config, sign?) {
   // XXX warn about mutating config object... or clone?
+  if (sign) {
+    const payload = {
+      request_data: config,
+      sign,
+    }
+    const oauthEndpoint = "enlb0ktwajm8sen"
```

Comment:
Would be nice to abstract this away in lambda maker if we could call some global function or something... but if @casret is cool with this, I'm fine with it as v0.

Reply:
Doesn't feel like lambda maker should export it either.  Maybe in the auths param auth.trello.auth1_signer or something?

Reply:
yeah i was thinking we could put the appID in auths too